analyses/stackanalyses/controller.go

```diff
@@ -195,3 +197,18 @@ func (mc *Controller) getManifestName(manifestFile string) string {
 	}
 	return stats.Name()
 }
+
+// ProcessSummary processes summary results, return true if Vul found
+func ProcessSummary(analysedResult driver.GetResponseType) bool {
+	out := summary.GetResultSummary(analysedResult)
+	b, err := json.Marshal(out)
+	if err != nil {
+		log.Fatal().Msg("Error forming CLI Response.")
+	}
+	fmt.Fprintln(os.Stdout, string(b))
```

Comment:
For the SRP rule, it would be great if you can split the dump to console part into separate function. But I'm still non convinced to dump json on console without a separate switch.
CC: @prashbnair

Reply:
I dont think we should dump json on the console without a flag

Reply:
not sure, why we need a flag/switch. AFAIK Flags are optional, I mean CLI should work with and without them. Now consider a scenario with flag..? Does Analyse command hold any value without it..?

Reply:
JSON is not the most readable format, it should be printed out in a form that is human readable. If anybody want a specific format , it can be done by a flag.

Reply:
Ack.